modules/ROOT/pages/bonita-bpm-studio-installation.adoc

```diff
@@ -190,3 +190,8 @@ A given Bonita Studio version can only run with some specifics Java versions (xr
 If you encounter some issues related to the JVM during the installation, ensure first that a valid Java version is available. You can for exemple type _java -version_ on a terminal, and informations about the main Java version used will be displayed. +
 If the problem persist, the easiest way to solve it is to remove the installed Java versions, to download that last stable build of the required Java version https://www.oracle.com/technetwork/java/javase/downloads/index.html[on the Oracle website] and to install it properly. +
 Those issues can be related to bad Java installations, or to incompatibilities between a given Java version and Bitrock (Bitrock is the install builder used to install Bonita, it has the responsibility to detect the JRE used to install Bonita. Some defects on a given version could lead to incompatibility).
+
```

Comment:
On row 143 (Troubleshooting), we would like to consistently identify the Troubleshooting sections starting in 2021.2.
This is why we use == image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
If you need to use it before 2021.2, you can add the image to this version too.